torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1011,10 +1013,10 @@ def method_tests():
         ('cumprod', (S, S, S), (1,), 'dim1', (), [0]),
         ('cumprod', (), (0,), 'scalar'),
         ('cumprod', lambda dtype, device: (torch.tensor(0., requires_grad=True, dtype=dtype, device=device)), (0,), 'scalar_zeros'),
-        #('cumprod', lambda dtype, device: prod_zeros(S, [0, 1], dtype), (1,), 'zeros_dim2', (), [0]),
-        #('cumprod', lambda dtype, device: prod_zeros(S, [0, 2], dtype), (1,), 'zeros_dim1', (), [0]),
-        #('cumprod', lambda dtype, device: prod_zeros(S, [1, 2], dtype), (1,), 'zeros_dim0', (), [0]),
-        #('cumprod', lambda dtype, device: prod_zeros(S, [1, 2], dtype), (1,), 'zeros_dim0_cast', (), [0], (), ident, {'dtype': torch.float64}),
+        ('cumprod', lambda dtype, device: prod_zeros(S, [0, 1], dtype), (1,), 'zeros_dim2', (), [0]),
+        ('cumprod', lambda dtype, device: prod_zeros(S, [0, 2], dtype), (1,), 'zeros_dim1', (), [0]),
+        ('cumprod', lambda dtype, device: prod_zeros(S, [1, 2], dtype), (1,), 'zeros_dim0', (), [0]),
+        ('cumprod', lambda dtype, device: prod_zeros(S, [1, 2], dtype), (1,), 'zeros_dim0_cast', (), [0], (), ident, {'dtype': torch.cdouble}),
```

Comment:
Should this be a new test to handle complex to keep both the double and cdouble tests?

Reply:
`double` test works only for non-complex types. Yeah, I see there a list that keeps only complex tests. I will add it there.